pkg/kubefed2/federate/federate.go

```diff
@@ -176,7 +193,7 @@ func lookupTypeDetails(config *rest.Config, qualifiedTypeName ctlutil.QualifiedN
 		return nil, errors.Wrapf(err, "Error retrieving API resource for FederatedTypeConfig %q", qualifiedTypeName)
 	}
 
-	glog.Infof("FederatedTypeConfig: %q found", qualifiedTypeName)
+	glog.V(2).Infof("FederatedTypeConfig: %q found", qualifiedTypeName)
```

Comment:
Please move these unrelated non-functional log changes to a separate commit.

Reply:
I think these changed the functionality, such that the yaml output in case of -o yaml becomes clean in the default verbosity case. I have anyways separated them into another commit.